README.md

```diff
@@ -1,52 +1,40 @@
-# n-header-footer [![Circle CI](https://circleci.com/gh/Financial-Times/n-header-footer/tree/master.svg?style=svg)](https://circleci.com/gh/Financial-Times/n-header-footer/tree/master)
+# o-header [![Circle CI](https://circleci.com/gh/Financial-Times/o-header/tree/master.svg?style=svg)](https://circleci.com/gh/Financial-Times/o-header/tree/master)
 
-Header and footer for next.ft.com sites
+Header for FT branded sites
 
 ## Installation
 
 ```
-bower install --S Financial-Times/n-header-footer
+bower install --S o-header
 ```
 
 ```js
-var headerFooter = require('n-header-footer');
+var oHeader = require('o-header');
 
-headerFooter.init();
+oHeader.init();
 ```
 
 ```scss
-@import 'n-header-footer/main';
+$o-header-is-silent: false;
+@import 'o-header/main';
 ```
 
-## Build
+## API
 
-To build the n-header-footer run
+config:
 
-```
-make build
-```
-
-This will also rebuild the navigation templates using data from `next-navigation`.
-
-## Feature flags
-
-To use feature flags pass in an instance of the next-feature-flags-client (which should already have been initialised) like so:
+* editionswitchClassName:
+* headerClassName:
 
-```js
-headerFooter.init(flags)`;
-```
-
-## Custom data
+## Data attributes
 
-When rendering templates the following can optionally be defined on the viewData
-
-```javascript
-headerData: {
-	isStream: Boolean,
-	section: {
-		url: '/blah/blah',
-		title: 'Blah Blah',
-		conceptId: 'taxonomy:&quot;name&quot;'
-	}
-}
-```
+* data-o-header-search
+* data-o-header--js
+* data-o-header-form-search
+    - input
+    - label
+* data-o-header-togglable-search
+* data-o-header-edition-switch-button
+* data-o-header-selectable
+* data-o-header-togglable
+* data-o-header-togglable-nav
```

Comment:
I haven't really worked on the documentation. I just added the stuff that I knew I would have to elaborate on so I wouldn't forget. I don't think I'll add the classes though, there are too many. For modules like this, we just suggest in the readme that people can check out the demo markup